tests/settings.py

```diff
@@ -9,7 +9,7 @@
 CURRENT_DIR = os.path.dirname(__file__)
 
 DEFAULT_NETWORK_CONFIGURATION_PATH = os.path.join(CURRENT_DIR, '../', 'values.yaml')
-SUBSTRAT_CONFIG_FILEPATH = os.getenv('SUBSTRAT_CONFIG_FILE', DEFAULT_NETWORK_CONFIGURATION_PATH)
+SUBSTRAT_CONFIG_FILEPATH = os.getenv('SUBSTRAT_CONFIG_FILEPATH', DEFAULT_NETWORK_CONFIGURATION_PATH)
```

Comment:
As it was discussed, we should change it to `SUBSTRA_*` ?

Reply:
In my opinion, this is outside the scope of this PR and changing `SUBSTRAT` to something else should be done in a separate PR.